tests/foundry/exclude

```diff
@@ -68,6 +68,9 @@ NonceTest.testNonce
 OwnerUpOnlyTest.testFailIncrementAsNotOwner
 OwnerUpOnlyTest.testIncrementAsNotOwner
 OwnerUpOnlyTest.testIncrementAsOwner
+PlainPrankTest.test_startPrankWithOrigin_true
+PlainPrankTest.test_stopPrank_true2
```

Comment:
Why are these tests excluded? Shouldn't they pass?

Reply:
I think so. Maybe they are there to be removed from the file when the last failing test passes. I'll try them out tomorrow.